libavcodec/qsvdec.c

```diff
@@ -748,6 +758,9 @@ static av_cold int qsv_decode_init(AVCodecContext *avctx)
         goto fail;
     }
 
+    if (!avctx->pkt_timebase.num)
```

Comment:
What's happened if just ignore msdk document just set avcodec pts to TimeStamp ?

Reply:
You might get wrong timestamp from the SDK and the muxer will print some error messge. e.g.

$> ffmpeg -hwaccel qsv -c:v hevc_qsv  -i output.h265  -f null -
[null @ 0x561c24f6f2f0] Application provided invalid, non monotonically increasing dts to muxer in stream 0: 2 >= 2